test/e2e/performanceprofile/functests/9_reboot/devices.go

```diff
@@ -171,6 +173,68 @@ var _ = Describe("[disruptive][node][kubelet][devicemanager] Device management t
 		Expect(err).ToNot(HaveOccurred(), "error checking the workload pod post reboot")
 		testlog.Infof("post reboot: newer workload pod %s/%s admitted: %s", updatedPod.Namespace, updatedPod.Name, extractPodStatus(updatedPod.Status))
 	})
+
+	It("Verify that pods requesting devices are not disrupted by a kubelet restart", func() {
+		namespace := testutils.NamespaceTesting
+
+		// refresh the targetNode object
+		node, err := testnodes.GetByName(targetNode)
+		Expect(err).ToNot(HaveOccurred(), "error getting the target node %q", targetNode)
+
+		// phase1: complete the node praparation: make sure we have enough devices. Short timeout, we should be idle now.
+		allocatableDevices := waitForNodeToReportResourcesOrFail("pre reboot", targetNode, sriovDeviceResourceName, 2*time.Minute, 2*time.Second)
+		Expect(allocatableDevices).To(BeNumerically(">=", minimumAllocatableDevices), "device %q has too low amount available - testing scenario unreliable", sriovDeviceResourceName, allocatableDevices)
```

Comment:
is it recommended here too that we have this amount of minimal devices? if I get it right, we don't want care about that the device plugin takes time to get back up, I actually think it's easier (less exec time) to get enough with only the workload limits. Is my understanding correct?

Reply:
good point. We can relax/remove this constraint for this test.